### DISCUSSION TITLE
EPW input files for Si example

### DISCUSSION THREAD

**Discussion by akentphonon** [nakib/elphbolt#58] [General]
Dear Nakib,

Thank you for this very useful code.

 I am trying reproduce Si example within elphbolt from scratch - by performing calculations for ifcs and electron-phonons elements with QE and EPW.  I have some doubts about EPW step -  what the specific tags needed to print electron-phonons elements within EPW calculations.  If possible would you please share EPW input file with me.

Thanks
Alex

**Comment by nakib** [comment]
Dear Alex,

Thank you for your interest in `elphbolt`!

I have provided two modified `EPW` source files in the repository. You will have to put these files in your `EPW/src` directory and rebuild `EPW`. Below I share the `EPW` input file that will print out all the Wannier space information that `elphbolt` requires. You can adapt it for your own calculation.

Note that `EPW` requires you to first do a phonon calculation with `Quantum Espresso`. Following that, you need to do a post-processing step with a script called `pp.py` that comes with `EPW`. You can look at the examples that ship with the `EPW` code and look at some tutorials to know more about how to do an `EPW` calculation. The `EPW` forum can give you more support on that.

Let me know if something is unclear. :) 

Best,
Nakib

epw.in:
```
&inputepw
  prefix = 'si'
  amass(1) = 28.086
  asr_typ = 'simple'

  outdir = '<enter epw output directory>'

  elph  = .t.
  kmaps = .f. 
  epbwrite = .t. 
  epbread = .f.
  etf_mem = 1
  epwwrite = .t. 
  epwread = .f. 

  nbndsub = 8

  wannierize  = .t. 
  num_iter = 1000
  dis_win_max = 17.5
  dis_froz_max = 9.5
  proj(1) = 'Si:sp3' ! 4 valence + 4 conduction bands

  wdata(1) = 'conv_tol = 1.0E-12'
  wdata(2) = 'dis_num_iter = 600'
  wdata(3) = 'dis_mix_ratio = 0.5d0'
  wdata(4) = 'dis_conv_tol = 1.0d-11'
  wdata(5) = 'num_print_cycles  = 50'

  elecselfen  = .false.
  phonselfen  = .false.
  a2f  = .false.

  dvscf_dir   = '<enter phonon "save" directory>'
 
  nk1         = 12
  nk2         = 12
  nk3         = 12
  nq1         = 6
  nq2         = 6
  nq3         = 6
 /
   16 cartesian
   0.000000000   0.000000000   0.000000000
  -0.166666667   0.166666667  -0.166666667
  -0.333333333   0.333333333  -0.333333333
   0.500000000  -0.500000000   0.500000000
   0.000000000   0.333333333   0.000000000
  -0.166666667   0.500000000  -0.166666667
   0.666666667  -0.333333333   0.666666667
   0.500000000  -0.166666667   0.500000000
   0.333333333   0.000000000   0.333333333
   0.000000000   0.666666667   0.000000000
   0.833333333  -0.166666667   0.833333333
   0.666666667  -0.000000000   0.666666667
   0.000000000  -1.000000000   0.000000000
   0.666666667  -0.333333333   1.000000000
   0.500000000  -0.166666667   0.833333333
  -0.333333333  -1.000000000   0.000000000
```

**Reply by akentphonon** [reply]
Thanks Nakib!

Your suggestions are very helpful. I will give them a go - I might bother you more queries later.

Thanks,
Alex
